src/sections/dataset/publish-dataset/PublishDatasetModal.tsx

```diff
@@ -60,7 +64,10 @@ export function PublishDatasetModal({
     )
     handleClose()
   }
-
+  if (releasedVersionExists && (!nextMajorVersion || !nextMinorVersion)) {
+    console.log('Error: nextMajorVersion or nextMinorVersion is missing')
+    return null
+  }
```

Comment:
I think we don't need this, if everything works well, this case should never happen.
and in any case we should not get to this point, if there is no released version and there is no major or minor version we should directly hide the button to publish dataset, but again I think we don't need that also.
What do you think?